Resources/Prototypes/Entities/Markers/Spawners/Random/maintenance.yml

```
@@ -382,3 +382,33 @@
         - Spaceshroom
       chance: 0.6
       offset: 0.0
+
+
+- type: entityTable
+  id: InsulsTable
+  table: !type:GroupSelector
+    children:
+    - id: ClothingHandsGlovesColorYellowBudget # budget insuls
+      weight: 85
+    - id: ClothingHandsGlovesColorYellow # true insuls
+      weight: 10
+    - id: ClothingHandsGlovesFingerlessInsulated # fingerless
+      weight: 4
+    - id: ClothingHandsGlovesConducting # conducting
+      weight: 1
+
+- type: entity
+  name: Maint Loot Spawner
+  id: MaintenanceInsulsSpawner
+  suffix: Insuls, safe
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Clothing/Hands/Gloves/Color/yellow.rsi
+          state: icon
+    - type: EntityTableSpawner
+      table: !type:NestedSelector
+        tableId: InsulsTable
+        prob: 1.0
```

```suggestion
    - type: Sprite
      layers:
      - state: red
      - sprite: Clothing/Hands/Gloves/Color/yellow.rsi
        state: icon
  - type: EntityTableSpawner
    table: !type:NestedSelector
      tableId: InsulsTable
```
[Yaml convention](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions) nitpick
Also 1 is the default probability, so we can omit that line

---

Can you give this another quick in-game test to see if I web-edited it correctly? I will merge after that.